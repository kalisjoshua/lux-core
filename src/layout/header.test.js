import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header';

describe('Default layout - header', function () {
  const component = renderer.create(
    <Header />
  );

  it('should match the snapshot', function () {
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
