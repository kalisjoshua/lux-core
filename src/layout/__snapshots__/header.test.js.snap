exports[`Default layout - header should match the snapshot 1`] = `
<header
  className="main-header"
  role="banner" />
`;
